TS02-Convert-to-Date-Time-Class-POSIX-In-R.Rmd

```diff
@@ -221,44 +224,51 @@ timeDatelt
 unclass(timeDatelt)
 ```
 
-When we converted the string to `POSIXlt`, it still appears to be the same as 
-`POSIXct`. However, `unclass()` shows us that the data are stored differently. 
-The `POSIXlt` class stores the hour, minute, second, day, month, and year as 
-separate components.
+When we convert a string to `POSIXlt`, and view it in `R`, it still looks similar to
+the `POSIXct` format. However, `unclass()` shows us that the data are stored differently. 
+The `POSIXlt` class stores the hour, minute, second, day, month, and year 
+separately.
+
+####Months in POSIXlt 
+Yet, `POSIXlt` has a few quirks. First, the month values stored in the `POSIXlt`
+object use `zero-based indexing`. This means that month #1 (January) is stored as
+0, and month #2 (February) is stored as 1. Notice in the output above, October is 
+stored as the 9th month (`$mon = 9`).
+
+####Years in POSIXlt 
+Years are also stored differently in the `POSIXlt` class. Year values are stored
+using a base index value of 1900. Thus, 2015 is stored as 115 (`$year = 115` 
+- 115 years since 1900).
 
-Yet, just like `POSIXct` being based on seconds since 1 Jan. 1970, `POSIXlt` has
-a few quirks.  First, the months use zero based indexing which starts counting
-with zero instead of 1, so October is stored as the 9th month (`$mon = 9`) and 
-the years are based on years since 1900, so 2015 is stored as 115 (`$year = 115`)
-Information like this can always be found in the `R` documentation of a package
-or function by using `?` (here, `?POSIXlt`).  
+<i class="fa fa-star"></i> **Data Tip:** To learn more about how `R` stores information
+within date-time and other objects AND also about functions, check out the 
```

Comment:
I'm confused by the "AND also about functions"  being included here.  You can't do ?functions and get any information.